src/app/base/components/sidebar-pm/sidebar-pm.component.ts

```diff
@@ -110,6 +112,17 @@ export class SidebarPmComponent implements OnInit, OnDestroy {
 
   }
 
+  private checkUserIsAdmin() {
+    if (!ConfigManager.isInit()) {
+      timer(250).subscribe(() => this.checkUserIsAdmin());
+      return;
+    }
+
+    if (ConfigManager.getIsAdmin()) {
+      this.isAdmin = true;
+    }
```

Comment:
`this.isAdmin = ConfigManager.getIsAdmin();`

Reply:
changed